dockers/docker-sonic-mgmt/Dockerfile.j2

```diff
@@ -94,6 +94,7 @@ RUN pip install allure-pytest==2.8.22 \
                 textfsm==1.1.3 \
                 thrift==0.11.0 \
                 virtualenv \
+                scandir \
```

Comment:
Can you insert the package in alphabetic order?

Reply:
Updated